pyproject.toml

```
@@ -26,7 +26,7 @@ python = ">=3.7,<4.0"
 tomli = { version = "^2.0.1", python = "<3.11" }
 click = "^8.0.0"
 importlib-metadata = { version = "*", python = "<=3.7" }
-chardet = "^5.0.0"
+chardet = "^4.0.0"
```

Per https://python-poetry.org/docs/dependency-specification/#caret-requirements, doing that will prohibit `>=5.x.y`, so it would be more suitable to use:
```suggestion
chardet = ">=4.0.0"
```

---

Thanks! I did not know that difference, learned something again. :)